testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpServerTest.groovy

```diff
@@ -352,12 +358,16 @@ abstract class HttpServerTest<SERVER> extends InstrumentationSpecification imple
 
   def "http pipelining test"() {
     assumeTrue(testHttpPipelining())
+    // test uses http 1.1
+    assumeFalse(useHttp2())
     expect:
     junitTest.httpPipelining()
   }
 
   def "non standard http method"() {
     assumeTrue(testNonStandardHttpMethod())
+    // test uses http 1.1
+    assumeFalse(useHttp2())
```

Comment:
why does this test need 1.1?

Reply:
Because the test uses netty to craft the request and sets the http version to 1.1 https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/facdc1826dee05bf534a07e3c281fcad846bea83/testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpServerTest.java#L492